platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapGestureDetector.java

```diff
@@ -428,7 +428,11 @@ public boolean onScale(ScaleGestureDetector detector) {
         transform.zoomBy(detector.getScaleFactor(), focalPoint.x, focalPoint.y);
       } else if (quickZoom) {
         // around center map
-        transform.zoomBy(detector.getScaleFactor(), uiSettings.getWidth() / 2, uiSettings.getHeight() / 2);
+        float scaleFactor = detector.getScaleFactor();
+        if (scaleFactor > 0.65 && scaleFactor < 1.35) {
```

Comment:
@tobrun I like the approach of limiting the values provided by the gesture library to handle this behavior. Two thoughts:

1. We're currently skipping updates when the `scaleFactor` is outside range. Wouldn't it work more smoothly if we `clamp` the value to the `[min ,max]` range?
2. Why are we choosing `0.65` and `1.35` as the cutoff values?